src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -171,7 +182,8 @@ def tokenizer_group(self) -> BaseTokenizerGroup:
 
     async def post_init(self) -> None:
         self.config = await self.engine.get_model_config()
-
+        # self.tokenizer_group = await self.engine.get_tokenizer_group()
+        self.tokenizer_group = self.engine.engine.tokenizer
```

Comment:
We should be able to use `self.engine.get_tokenizer_group()`:

```suggestion
        self.tokenizer_group = self.engine.get_tokenizer_group()
```

This has been in since [0.4.3](https://github.com/vllm-project/vllm/blob/v0.4.3/vllm/engine/llm_engine.py#L381)

We'll have to bump the minimum vLLM version in `pyproject.toml` aswell.

Reply:
+1, I thought we had updated that in ibm/vllm main but looks like we didin't (but should)

Reply:
Technically it is `self.engine.engine.get_tokenizer_group()` since `engine.engine` is `_AsyncLLMEngine` which is the class that derives from `LLMEngine` that has the `get_tokenizer_group` function